### DISCUSSION TITLE
How to remove the wavelength markers around the edge of the spectral locus?

### DISCUSSION THREAD

**Discussion by hminle** [colour-science/colour#741] [Q&A] [answered]
Hi Colour Team,

How can we remove the wavelength markers around the edge of the border of CIE diagram?
Those 540, 520, ....
![image](https://user-images.githubusercontent.com/16201681/80282773-53357f80-86e1-11ea-8f72-f0e694ce081c.png)

Thanks,

**Comment by hminle** [comment, marked as answer]
oh I found it,
Just add this parameter
```show_spectral_locus=False```

**Comment by thomasmansencal** [comment]
@hminle : Alternatively you can pass `spectral_locus_labels=[]`, it will still draw the spectral locus but without any markers: https://github.com/colour-science/colour/blob/817b1aa9a7456cf4c37e93adbf374b950d4cb0cf/colour/plotting/diagrams.py#L71
